database/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:16.04
```

Comment:
@djophant remove the folder database before merge, this feat(insta dependencies) must only include code about instagram and not docker artifacts